### ISSUE TITLE
RetCode:8378. Message:if you want to resize disk, please use ResizeAttachedDisk

### ISSUE THREAD

**Issue by ushuz** [ucloud/ucloud-cli#45]
```
$ ucloud uhost resize --uhost-id uhost-xxx/xxx --data-disk-size-gb 500
Something wrong. RetCode:8378. Message:if you want to resize disk, please use ResizeAttachedDisk
```

**Comment by lixiaojun629** [comment]
感谢反馈问题，我们尽快在下个版本修复。

**Comment by lixiaojun629** [comment]
fix by #46
